### DISCUSSION TITLE
Foundry Complier Error (The application panicked (crashed)

### DISCUSSION THREAD

**Discussion by moviemanrico** [Cyfrin/foundry-full-course-cu#1152] [General]
Hi guys,
I'm having this message when I'm compiling a contract. I need help.

' ' '
The application panicked (crashed).
Message:  byte index 59 is not a char boundary; it is inside 'ğ' (bytes 58..60) of `5 | import {script} from "Users/cannd/Desktop/BlockChain-Eğitim/foundry-f23/forge-std-master/src/script.sol";`
Location: /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/foundry-compilers-0.1.1/src/artifacts/mod.rs:2085

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Aborted
' ' '

**Comment by 0xusmanf** [comment]
The problem seems to be in the path name where you saved your project due to the fact Foundry is unable to recognize the special character `ğ`.

Renaming it to a simple character might work.
